### DISCUSSION TITLE
Tailwind not compiling new classes

### DISCUSSION THREAD

**Discussion by EnzoBDF** [tailwindlabs/tailwindcss#14014] [Help]
I'm doing a project with Express and Tailwind, and after sometime, i've noticed that tailwind wasn't compiling new utility classes, but only others who were already being used. Did someone already had this problem or know how to solve this. Any help would be really appreciated

**Comment by wongjn** [comment]
Consider checking:
- The `content` file globs sufficiently cover the files that code appears in.
- [String concatenation/interpolation is not used to build class names](https://tailwindcss.com/docs/content-configuration#dynamic-class-names).

**Reply by EnzoBDF** [reply]
Actually, i don't have a file with such name, if you want i can put a scresnhot of how the project is structured and the css/tailwind/postcss files

**Reply by wongjn** [reply]
I'm not sure how your project is structured, but consider reviewing [a framework guide](https://tailwindcss.com/docs/installation/framework-guides) if any of them are relevant, or [the PostCSS integration guide](https://tailwindcss.com/docs/installation/using-postcss). Furthermore, consider reviewing the [Content configuration section in the documentation](https://tailwindcss.com/docs/content-configuration).

**Reply by EnzoBDF** [reply]
Gonna do it, thank you for the help and time
